src/services/mongodb/collection.js

```diff
@@ -143,109 +143,56 @@ Collection.prototype.upsert =
 
```

Comment:
let's take away `upsert`. this is a good opportunity to remove deprecated methods, so we should go for it. Then we won't need to import `deprecate` anymore, either.

Reply:
It looks like  `getCollection` and `getDB`/`getDb` can get removed too (in `src/services/mongodb/db.js` and `src/services/mongodb/mongodb_service.js`).

Reply:
FYI: there was an unused `insert` function (that sat alongside `upsert`) that I removed as well (it just wasn't being deprecated for whatever reason)